src/view/wait.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { StyleSheet, ActivityIndicator } from 'react-native';
+import Background from '../component/background';
+import MainContent from '../component/main-content';
+import { color } from '../component/style';
+
+const styles = StyleSheet.create({
+  content: {
+    justifyContent: 'center',
+  },
+  spinner: {
+    transform: [{ scale: 1.5 }],
+  },
+});
+
+const WaitView = () => (
+  <Background color={color.blackDark}>
+    <MainContent style={styles.content}>
+      <ActivityIndicator
+        size="large"
+        color={color.lightPurple}
+        style={styles.spinner}
+      />
```

Comment:
I might add a `copy` section with a small description of what the user is waiting on.

Reply:
Would make sense if we used it only on one screen. We're going to replace this anyway with a fancier design anyway. So I think it's ok for now.